lasagne/layers/recurrent.py

```diff
@@ -564,9 +550,9 @@ def get_output_for(self, input, mask=None, *args, **kwargs):
         # Because the input is given for all time steps, we can precompute
         # the inputs to the gates before scanning.
         # input is dimshuffled to (n_time_steps, n_batch, n_features)
-        # W_in_to_gates is (n_features, 4*num_units). input_dot_W is then
+        # W_in_to_gates is (n_features, 4*num_units). input_dot_w is then
```

Comment:
Its pycharns PEP8. I don't know if it is correct.
![pep8](https://cloud.githubusercontent.com/assets/2623134/7677376/cc4af1ea-fd4b-11e4-8309-12518d9b2db6.png)